### PR TITLE
Ei 429 new collection function project

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess
+{
+    public static class IncentivePaymentOrchestrator
+    {
+        [FunctionName("IncentivePaymentOrchestrator")]
+        public static async Task RunOrchestrator([OrchestrationTrigger] IDurableOrchestrationContext context, ILogger log)
+        {
+            var collectionPeriod = context.GetInput<string>();
+
+            log.LogInformation($"Incentive Payment process started for collection period {collectionPeriod}", new { collectionPeriod });
```
Comment (Stephek) [inline src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs:15]: If you delegate the Orchestrator call to a service then you can add the logging as a decorator including logging on error keeping the concerns of logging separate from the functionality.

**Review by Stephek — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/.gitignore [hunk 2]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/.gitignore:1]: is this needed?

File: src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/.gitignore [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+# JetBrains Rider
+.idea/
+*.sln.iml
+
+# CodeRush
+.cr/
+
+# Python Tools for Visual Studio (PTVS)
+__pycache__/
+*.pyc
```
Comment (ruslanss) [inline src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/.gitignore:264]: is **this** needed? 😁

Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/.gitignore]: No, not at all.  I've deleted it.

File: src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs (hunk 1 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs:15]: I'm not sure how practical delegating the orchestration would be as you'd need to pass the DurableOrchestrationContext around...I'll stick something in the dev channel.

Comment (Stephek) [reply src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/IncentivePaymentOrchestrator.cs:15]: Ok

**Review by David-B-Read — APPROVED** [review]

**Review by ruslanss — APPROVED** [review]

**Review by Stephek — APPROVED** [review]

**Pull request closed without merging**